files/build_templates/init_cfg.json.j2

```diff
@@ -83,6 +83,20 @@
             "state" : "disabled", {% endif %}
             "rate_limit_interval" : "600"
         }{%if not loop.last %},{% endif -%}
-{% endfor %} 
+{% endfor %}
+    },
+    "PASSW_HARDENING": {
```

Comment:
Shouldn't this table be present only when feature flag is turned on? Can you please add the j2 check for it?